tools/kagami/README.md

```diff
@@ -2,206 +2,12 @@
 
 Kagami is a tool used to generate and validate automatically generated data files that are shipped with Iroha.
 
-## Build
```

Comment:
I would leave this Build section to show and to explain how we build binary separately. And in Usage section I would leave the following:
```bash
kagami --help
```